cli/cmd/uninstall.go

```diff
@@ -36,9 +36,16 @@ var uninstallCmd = &cobra.Command{
 		}
 		ctx := cmd.Context()
 
+		errMessage := fmt.Errorf("Odigos is not installed in the cluster")
```

Comment:
This variable should be declared in the internal `if` next to where it is being used.